build.gradle

```diff
@@ -31,10 +31,13 @@ buildscript {
 
```

Comment:
Cant figure out how to add a comment to the previous lines, but the comment on line 17 should be updated.

Reply:
Done